wlog/svclog/svc1log/logger_default.go

```diff
@@ -15,9 +15,37 @@
 package svc1log
 
 import (
+	"time"
+
 	"github.com/palantir/witchcraft-go-logging/wlog"
 )
 
+var debugLevelParam = wlog.NewParam(func(entry wlog.LogEntry) {
```

Comment:
Is the declaration of these variables as top-level variables rather than inlining as part of the function return intentional for purposes of performance/allocation? If so, then that's fine, but would add comment that explains this and then declare these in block form:

```go
var (
   debugLevelParam = ...
   infoLevelParam = ...
   // etc.
)
```

If there isn't a performance-related or otherwise intentional reason for these declarations, then would prefer that they not be declared as variables and inline in the functions instead.